### PR TITLE
Intake

### PR DESCRIPTION
**Pull request by Sounder317**
Ready to merge got talon working.

### CODE REVIEW

**Review by brycecary — CHANGES_REQUESTED** [review]
Please fix your formatting according to the below standards:

https://cavineers.github.io/CodeStandards/docs/java-indentation.html

**Comment by Sounder317** [comment]
I think I fixed the formatting

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// TODO: DrumMotor
+// TODO: DrumMotor fix
 
 package frc.robot.subsystems;
```
Comment (brycecary) [inline src/main/java/frc/robot/subsystems/Intake.java:2]: This is still showing 2 space indentation for me. If you don't mind switching it to 4 spaces.
https://cavineers.github.io/CodeStandards/docs/java-indentation.html

**Review by brycecary — CHANGES_REQUESTED** [review]
1. Can you fix the spacing \([Reference](https://cavineers.github.io/CodeStandards/docs/java-indentation.html)\)
1. Can you remove OI from the imports, a subsystem shouldn't need to call it
1. `ControlMode.PercentOutput` must be between the values of [-1, 1] \([Reference](http://www.ctr-electronics.com/downloads/api/java/html/enumcom_1_1ctre_1_1phoenix_1_1motorcontrol_1_1_control_mode.html#adc64ac712cda8e44241a9c476dbff22e)\)
1. The logic seems a little much. Try simplifying it if you can.

**Pull request closed without merging**